feature/search/api/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/search/api/navigation/SearchNavigation.kt

```diff
@@ -33,13 +40,13 @@ fun NavGraphBuilder.searchScreen(
     onInterestsClick: () -> Unit,
     onTopicClick: (String) -> Unit,
 ) {
-    // TODO: Handle back stack for each top-level destination. At the moment each top-level
-    // destination may have own search screen's back stack.
-    composable<SearchRoute> {
-        SearchRoute(
-            onBackClick = onBackClick,
-            onInterestsClick = onInterestsClick,
-            onTopicClick = onTopicClick,
-        )
-    }
+//    // TODO: Handle back stack for each top-level destination. At the moment each top-level
+//    // destination may have own search screen's back stack.
```

Comment:
Maybe we should just remove the search option from all top level destinations except "For You".

Reply:
Right now with Nav3 backstack, they all share the same search backstack. And it feels right to have it on `Interests` as well.